python/tvm/relay/op/strategy/x86.py

```diff
@@ -96,6 +99,12 @@ def conv2d_strategy_cpu(attrs, inputs, out_type, target):
                     wrap_compute_conv2d(topi.x86.conv2d_nchw),
                     wrap_topi_schedule(topi.x86.schedule_conv2d_nchw),
                     name="conv2d_nchw.x86")
+        if layout == "NCHW":
+            assert kernel_layout == "OIHW"
+            add_implementation_nchw()
+        elif _NCHWc_matcher.match(layout): # check if layout is NCHWxc
+            assert _OIHWio_matcher.match(kernel_layout) # check if kernel is OIHWio
+            add_implementation_nchw()
```

Comment:
Here you should directly add `topi.x86.conv2d_NCHWc` since the op already has NCHWc layout.

Reply:
Ok. now redirected to `conv2d_NCHWc_strategy_cpu`